.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -33,9 +26,24 @@ Make sure to open and fill in a separate **documentation issue** in your product
 > [!NOTE]
 > Note: most criteria for documentation and security are now covered in TRGs
 
-# Security Checks
+# Features
+Please provide a list of all features that have been developed in the current phase. This list is essential for several reasons:
+
+- **Test Preparation:** Identifying new features will help in the creation of new end-to-end (E2E) tests and ensure that the relevant aspects are thoroughly tested.
+- **Awareness:** It will raise awareness of the features that need to be prioritized during the testing process.
+- **Board Management:** This will assist in keeping the project board organized and up-to-date.
+
+By documenting these features, you will contribute to a smoother and more efficient release process.
+
+List your features here:
+
+- Link to feature
+
+> [!NOTE]
+> Note: all features needs to be closed to get approval
 
-- [ ] **Threat Modeling Analysis** passed
+# Summary
```

Comment:
Please add empty line for proper md format

Reply:
done :)